stock_product_qty_by_packaging/tests/test_stock.py

```diff
@@ -0,0 +1,63 @@
+# Copyright 2021 Camptocamp SA
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl)
+from odoo.addons.stock_packaging_calculator.tests.common import TestCommon
```

Comment:
To fix once https://github.com/OCA/stock-logistics-warehouse/pull/2294 is merged.

```suggestion
from odoo.addons.product_packaging_calculator.tests.common import TestCommon
```